### DISCUSSION TITLE
Citing MoreLINQ

### DISCUSSION THREAD

**Discussion by UIowaJinCho** [morelinq/MoreLINQ#1071] [Q&A] [answered]
Hello,

To whom it may concern:

I recently developed some research software using MoreLINQ and I was wondering if it has specific guidelines for citation for a journal publication.

Thank you very much for your time.

**Comment by atifaziz** [comment]
Thank you for reaching out to ensure proper citation of open source software. This is the first ask of this nature and so I don't have specific guidelines to share, especially when it comes to a journal publication. Do you have a citation proposal that I can help review?

**Comment by atifaziz** [comment, marked as answer]
@UIowaJinCho Does the following help?

> Aziz, A., Leandro, F. V., A., Sateler, F., Bushkin L., Skeet, J. et al. (2024). MoreLINQ _version,_ version : LINQ to Objects, extended. https://github.com/morelinq/MoreLINQ

Replace _version_ with actual version used.

**Reply by UIowaJinCho** [reply]
I think this is it! Thank you very much! I did not realize that you responded super quick. Sorry for the late response.
